crates/bevy_ecs/src/lib.rs

```diff
@@ -1228,9 +1342,7 @@ mod tests {
         let (dropck1, dropped1) = DropCk::new_pair();
```

Comment:
This is no longer a sparse drop check / it is now redundant with the previous test. We should use a DropCkSparse component here.